content/articles/how-to-plot-data-in-python-using-plotly/index.md

```diff
@@ -0,0 +1,145 @@
+0   Afghanistan     Asia    1952    28.801  8425333     779.445314  AFG     4
+1   Afghanistan     Asia    1957    30.332  9240934     820.853030  AFG     4
+2   Afghanistan     Asia    1962    31.997  10267083    853.100710  AFG     4
+3   Afghanistan     Asia    1967    34.020  11537966    836.197138  AFG     4
+4   Afghanistan     Asia    1972    36.088  13079460    739.981106  AFG     4
+...     ...     ...     ...     ...     ...     ...     ...     ...
+1699    Zimbabwe    Africa  1987    62.351  9216418     706.157306  ZWE     716
+1700    Zimbabwe    Africa  1992    60.377  10704340    693.420786  ZWE     716
+1701    Zimbabwe    Africa  1997    46.809  11404948    792.449960  ZWE     716
+1702    Zimbabwe    Africa  2002    39.989  11926563    672.038623  ZWE     716
+1703    Zimbabwe    Africa  2007    43.487  12311143    469.709298  ZWE     716
+```
+
+By looking at these numbers, it isn't easy to see what this data is telling us. It's hard to comprehend. So now, let's create the data visualization using interactive animations to help us visualize this data to understand it better.
+
+```python
+px.scatter(data_frame=gapminder_df,
+            x = 'gdpPercap',
+            y = 'lifeExp',
+            size = 'pop',
+            color = 'continent',
+            title = 'Life Span and Wealth 1952 - 2007',
+            labels = { 'gdpPercap': 'Wealth',
+                        'lifeExp': 'Life Span'}, 
+            log_x = True,
+            range_y = [25,95],
+            hover_name = 'country',
+            animation_frame = 'year',
+            height = 600,
+            size_max = 100)
+```
+To summarize this code:
+
+-  `x = 'gdpPercap'` and  `y = 'lifeExp'` tells us what we want our x and y coordinates to be. The x-axis will represent the gdpPercap which has been assigned the label `Wealth`, while the y axis will represent the life expectancy assigned to the label `Life Span`.
+- `color = 'continent',` tells plotly that we want each continent to have a different color.
+- `title = 'Life Span and Wealth 1952 - 2007',` is the title of our visualization. You can change it to whichever title fits you best.
+- `log_x = True,` converts the scale on the x-axis to be a log scale the distribution as it is is quite wide.
+- `range_y = [25,95],` constrains the range on the y-axis to between 25 and 95 years
+- `hover_name = 'country',` tells us that when we hover our mouse on the circles, we want it to tell us which country it represents. 
+- `height = 600,` tells plotly to adjust the size of the plot to 600px and `size_max = 100` to adjust the size of each point to 100px.
+
+Now, let's take a look at our output when we run the code.
+
+![Output](/engineering-education/how-to-plot-data-in-python-using-plotly/plotly-output-52.png)
+
+The above image shows the distribution of wealth in the year 1952.
+
+![Output](/engineering-education/how-to-plot-data-in-python-using-plotly/plotly-output-07.png)
```

Comment:
Consider explaining how a reader is meant to interpret the results of this plot to add value